src/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -5,6 +5,8 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.ComponentModel;
```

Comment:
We don't use using statements in reference assembly. Instead use the full name in the type. i.e:

```cs
System.ComponentModel.TypeConverter
```